cmd/kube-scheduler/app/server.go

```diff
@@ -155,6 +155,8 @@ func NewOptions() (*Options, error) {
 	// TODO: we should fix this up better (PR 59732)
 	o.config.LeaderElection.LeaderElect = true
 
+	o.policyConfigMapNamespace = metav1.NamespaceSystem
```

Comment:
Move to L142?

Reply:
prefer to give a default value of `policy-configmap-namespace`; so user knows which namespace will be used by default.

Reply:
```
➜  kube-scheduler git:(kube-schedule-policy-configmap-namespace) ✗ ./kube-scheduler -h | grep policy-configmap-namespace       
      --policy-configmap-namespace string      The namespace where policy ConfigMap is located. The system namespace will be used if this is not provided or is empty. (default "kube-system")
```

@k82cn cobra did it~

Reply:
I mean set default value by cobra, update the following line.

```
fs.StringVar(&o.policyConfigMapNamespace, "policy-configmap-namespace", o.policyConfigMapNamespace, "The namespace where policy ConfigMap is located. The system namespace will be used if this is not provided or is empty.")
```

Reply:
Seems like it would be better to make the namespace explicit

Reply:
ref https://github.com/kubernetes/kubernetes/issues/60366
from @dixudx 's this issue, i think use  o.policyConfigMapNamespace would be better.